tests/cpp/test_circular_buffering.cpp

```diff
@@ -800,24 +804,43 @@ TEST_P(CircularBufferingTest, NoSync) {
 }
 
 // Test circular buffer from 2 to 10 stages
+auto StagesAndPrefetches() {
+  std::vector<StageAndPrefetch> values;
+  for (int64_t i : c10::irange(2, 10)) {
+    for (int64_t j : c10::irange(i)) {
```

Comment:
Should we test some negative prefetch values?

Reply:
Good idea. Added.